src/App.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
-import { Container, Typography } from '@mui/material';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import LandingPage from './view/index';
+import Login from './view/login';
 
 const App = () => (
-  <Container fullWidth>
-    <Typography variant="h3">Barefoot Nomad</Typography>
-  </Container>
+  <BrowserRouter>
+    <Routes>
+      <Route exact path="/" element={<LandingPage />} />
+      <Route exact path="/login" element={<Login />} />
+    </Routes>
+  </BrowserRouter>
 );
```

Comment:
Create a folder called `routes` which will have all routes; and add this file

Reply:
I have created that folder as requested.